src/main/java/com/minecolonies/coremod/client/gui/GuiFactory.java

```diff
@@ -35,7 +35,7 @@ public void initialize(Minecraft minecraftInstance)
      */
     @Override
     @Deprecated
-    public RuntimeOptionGuiHandler getHandlerFor(RuntimeOptionCategoryElement element)
+    public RuntimeOptionGuiHandler getHandlerFor(final RuntimeOptionCategoryElement element)
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1133)